codemirror-ls/src/LSPlugin.ts

```diff
@@ -249,3 +287,13 @@ export class LSCore extends LSCoreBase implements PluginValue {
 }
 
 export const LSPlugin = ViewPlugin.fromClass(LSCore);
+
+/**
+ * Error thrown when a lock could not be acquired within a certain timeout.
+ */
+export class LSLockTimeoutError extends LSError {
```

Comment:
Probably belongs in the `errors.ts` file along with the rest of the error types?